### PR TITLE
Enable admission_type filter and default to 'ALL'

### LINKED ISSUE #829: Enable default filter for admission_type='All' to prepare for admission type data to land
**What needs to be done? Why does it need to be done?**
Set default admission_type filter to 'ALL' and enable filtering to prepare for admission type data to land - prior to enabling the filter component on the PA dashboard.

**Additional context**
Add any other context about the task here.

### PR DESCRIPTION
**Pull request by jovergaag**
## Description of the change

Enables the `admission_type` filter with default 'ALL' value to support admission_type data landing in staging.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #829

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/RootStore/__tests__/BaseDataStore.test.js [hunk 1]
```diff
@@ -272,8 +270,9 @@ describe("BaseDataStore", () => {
 
     it("fetches a new subset file with new filter query params", () => {
       const expectedEndpoint = `${tenantId}/newRevocations/revocations_matrix_distribution_by_district?
-      metricPeriodMonths=12&chargeCategory=All&reportedViolations=All&violationType=LAW&
-      supervisionType=All&supervisionLevel=All&levelOneSupervisionLocation[0]=All&levelTwoSupervisionLocation[0]=All`.replace(
+      metricPeriodMonths=12&chargeCategory=All&reportedViolations=All&violationType=All&
+      supervisionType=All&supervisionLevel=All&levelOneSupervisionLocation[0]=All&
```
Comment (daschi) [inline src/RootStore/__tests__/BaseDataStore.test.js:274]: I think we would also want to add `admissionType` to the params validations too on the BE?

Comment (jovergaag) [reply src/RootStore/__tests__/BaseDataStore.test.js:274]: Good call.

**Comment by daschi** [comment]
Should I be reviewing https://github.com/Recidiviz/pulse-dashboard/pull/850 instead?

**Comment by jovergaag** [comment]
> Should I be reviewing #850 instead?

This needs to be deployed first to prepare for the admission_type data to land in production. Then the filter itself can be merged and deployed next week once the data is ready. 850 is based on this branch but we want to merge this first so that we don't have to coordinate 850 with the prod deploy.

**Comment by jovergaag** [comment]
@daschi Can you please take another look at this? I am hoping to deploy prod tomorrow and we need to get this one in there in preparation for Tuesday!

Thank you!

**Review by daschi — APPROVED** [review]

**Pull request merged**